src/pyconverter/xml2py/writer.py

```diff
@@ -239,14 +241,17 @@ def write_source(
     xml_doc_path : str
         Path containing the XML directory to convert.
 
-    template_path : str
-        Path containing the ``_package`` directory.
+    target_path : str
+        Path to generate the new package to.
 
     path_custom_functions : str, optional
         Path containing the customized functions. The default is ``None``.
 
-    new_package_path : str, optional
-        Path to copy the ``_package`` directory to. The default is ``./package``.
+    new_package_name : str, optional
+        Name of the new package. The default is ``package``.
+
+    template_path : str, optional
+        Path for the template to use. If no path is provided, the default template is used.
```

Comment:
Reorder. The function's definition has these arguments the other way around
```suggestion
    template_path : str, optional
        Path for the template to use. If no path is provided, the default template is used.

    new_package_name : str, optional
        Name of the new package. The default is ``package``.
```